subscription-engine/kafka-ws/Dockerfile

```diff
@@ -45,5 +45,5 @@ ENTRYPOINT ["/sbin/tini", "--"]
 CMD ["./bin/entryPoint.sh", "npm", "run", "kafka-ws"]
 
 HEALTHCHECK --start-period=5s --interval=30s --timeout=5s --retries=2 \
-    CMD HC=$(wget -q -O - "http://localhost:${KAFKA_WS_PORT:-8080}/healthcheck") \
-        && echo $HC && echo $HC | grep -q '"status":"ok"'
\ No newline at end of file
+    CMD HC=$(wget -q -O - "http://localhost:9000/health") \
```

Comment:
To simplify you can change to  `HEALTHCHECK --start-period=2m --interval=30s --timeout=10s --retries=3 \
    CMD curl -f http://localhost:9000/health || exit 1`


https://github.com/dojot/dojot-microservice-sdk-js/blob/master/examples/serviceStateManager/Dockerfile#L42